### ISSUE TITLE
First Note request for Explainer for W3C Accessibility Guidelines (WCAG) 3.0

### ISSUE THREAD

**Issue by michael-n-cooper** [w3c/transitions#394]
# Document title, URLs, estimated publication date
title: Explainer for W3C Accessibility Guidelines (WCAG) 3.0
editors' draft uri: https://w3c.github.io/silver/explainer/
estimated publication date: 2 December 2021
requested latest version uri: https://www.w3.org/TR/wcag-3.0-explainer/
anticipated publication uri: https://www.w3.org/TR/2021/WD-wcag-3.0-explainer-20211202/

# Abstract

This Explainer accompanies the drafts of the W3C Accessibility Guidelines (WCAG) 3.0. 

# Status of the Document

This section describes the status of this document at the time of its publication. A list of current W3C publications and the latest revision of this technical report can be found in the W3C technical reports index at https://www.w3.org/TR/.
This is a First Public Working Draft of Explainer for W3C Accessibility Guidelines (WCAG) 3.0 by the Accessibility Guidelines Working Group together with the Silver Task Force and Silver Community Group. It describes the structure of W3C Accessibility Guidelines (WCAG) 3.0, providing important introductory content for readers of that document. This version includes content that was previously published in drafts of WCAG 3.0, updated since then in this document.

This document was published by the Accessibility Guidelines Working Group as a Working Draft using the Recommendation track.

Publication as a Working Draft does not imply endorsement by W3C and its Members.

This is a draft document and may be updated, replaced or obsoleted by other documents at any time. It is inappropriate to cite this document as other than work in progress.

This document was produced by a group operating under the 1 August 2017 W3C Patent Policy. W3C maintains a public list of any patent disclosures made in connection with the deliverables of the group; that page also includes instructions for disclosing a patent. An individual who has actual knowledge of a patent which the individual believes contains Essential Claim(s) must disclose the information in accordance with section 6 of the W3C Patent Policy.

This document is governed by the 2 November 2021 W3C Process Document.

To comment, file an issue in the W3C silver GitHub repository. The Working Group requests that public comments be filed as new issues, one issue per discrete comment. It is free to create a GitHub account to file issues. If filing issues in GitHub is not feasible, send email to public-agwg-comments@w3.org (comment archive). The Working Group requests comments on this draft be sent by 28 January 2022. In-progress updates to the guidelines can be viewed in the public editors' draft.

# Link to group's decision to request transition

https://lists.w3.org/Archives/Public/w3c-wai-gl/2021JulSep/0171.html

# Is it intended to be a W3C Statement?

No

**Comment by plehegar** [comment]
Your SOTD talks about "This is a First Public Working Draft of Explainer for W3C ". Working Drafts cannot be Notes anymore.

Use ReSpec [status](https://respec.org/docs/#specStatus) "DNOTE" (Draft Note) instead.

Approved, with the status fix.

**Comment by michael-n-cooper** [comment]
[Publication request](https://lists.w3.org/Archives/Team/webreq/2021Dec/0007.html)
